deployment/v3/external/msg-gateway/install.sh

```diff
@@ -15,29 +15,37 @@ function msg_gateway() {
   echo Istio label
   kubectl label ns $NS istio-injection=enabled --overwrite
 
-  HOST=mock-smtp.mock-smtp
-  PORT=8025
-  USER=
-  SECRET="''"
+  SMTP_HOST=mock-smtp.mock-smtp
+  SMS_HOST=mock-smtp.mock-smtp
+  SMTP_PORT=8025
+  SMS_PORT=8080
+  SMTP_USER=
+  SMS_USER=
+  SMTP_SECRET="''"
+  SMS_SECRET="''"
 
   read -p "Would you like to use mock-smtp (Y/N) [ Default: Y ] : " yn
   # Set yn to N if user input is null
   if [ -z $yn ]; then
     yn=Y;
   fi
   if [ $yn != "Y" ]; then
-      read -p "Please enter the SMTP host " HOST
-      read -p "Please enter the SMTP host port " PORT
-      read -p "Please enter the SMTP user " USER
-      read -p "Please enter the SMTP secret key " SECRET
+      read -p "Please enter the SMTP host " SMTP_HOST
```

Comment:
We need to seperate mock-smtp and mock-sms prompts.
This is bacause there could be customers who have their own smtp channel and not the sms. Hence they will be able to decide easily.